Gemfile

```diff
@@ -1,6 +1,6 @@
 #
 # Please keep this file alphabetized and organized
 #
-source 'http://rubygems.org'
+source 'https://rubygems.org'
```

Comment:
If you're making this change here, you might want to make it in `gemfiles/nokogiri-1.5.gemfile` as well.